### ISSUE TITLE
Example does not make sense

### ISSUE THREAD

**Issue by constantinpape** [inferno-pytorch/inferno#147]
I just checked the example we have in the README and I think it does
not make sense....
We are adding a `Softmax` and use `CrossEntropyLoss` (which combines `Softmax` and `NLLLoss`).

**Comment by constantinpape** [comment]
Fixed in 28d7c16e2cbd7c186f642bc5b8883d48d2a1294c
